backend/audit/etl.py

```diff
@@ -122,9 +133,10 @@ def load_federal_award(self):
                 award_reference=entry["award_reference"],
                 federal_agency_prefix=program["federal_agency_prefix"],
                 federal_award_extension=program["three_digit_extension"],
-                additional_award_identification=program[
-                    "additional_award_identification"
-                ],
+                # CONDITIONAL
+                additional_award_identification=self.conditional_lookup(
```

Comment:
If a helper to do conditional lookups in the JSON object is going to be used, use it consistently everywhere. Or, don't use it.